src/wallet/walletdb.h

```diff
@@ -41,6 +41,24 @@ enum DBErrors
     DB_NEED_REWRITE
 };
 
+class CWalletScanState {
```

Comment:
Splitting it up across files is pretty ugly in that case. Is there a next step PR in the pipeline to fix that?